doc/custom-readers.md

```diff
@@ -27,14 +27,19 @@ which is automatically in scope.  (Indeed, all of the utility
 functions that are available for [Lua filters] are available
 in custom readers, too.)
 
+Each input source contains the *text*, and the *name* of an input
+file as a property of the respective name. The input sources
```

Comment:
I find this a bit hard to understand.  "the *name* of an input file as a property of the respective name".  An example could help?